### DISCUSSION TITLE
NBA裁判在亂吹什麼！

### DISCUSSION THREAD

**Discussion by a3240281370** [1101-GS4719-C/Python-Community-GS4719-C#1406] [General]
最近NBA季後賽第二輪如火如荼地開打，先不論是支持哪隊的，只要比賽都會有一些具爭議性的吹判，而這些爭議的吹判在關鍵時刻更有可能影響比賽局勢，於是我就在想不論是籃球或是棒球羽球等等，為何不全面改用機器判決呢？
註：目前許多的球類已經有電腦的影像輔助判決，但通常會有次數或其他規則的限制，主要還是以裁判判決為主
以下是我想到的判決優點和缺點：
優點：
1. 公平性提升：減少人為錯誤。這有助於提高比賽的公平性，避免由於主觀判斷而引起的爭議。
2. 減少爭議：幫助裁判員解決爭議，特別是在重大決定或關鍵時刻。這有助於減少爭議和不公平的感覺，提高觀眾和參與者對比賽結果的接受度。

弊端：
1. 時間消耗：可能導致比賽的節奏受到影響，並可能引起觀眾的不滿。
2. 削弱人的因素：有些人可能會覺得比賽因此變得冷冰冰，失去了人類判斷的情感和動力。

大家比較希望這類比賽是改用全面機器判決還是一樣以人為判決為主呢？

**Comment by PaulSu0905** [comment]
但機器判決失去了人類判斷的情感和動力。人類裁判的經驗、專業知識和判斷力在某些情況下可能無法被機器完全取代。例如違反運動精神的犯規很難只用機器做判斷。
